### DISCUSSION TITLE
db provider as environment variable

### DISCUSSION THREAD

**Discussion by Shahzayb** [prisma/prisma#3101] [Q&A] [answered]
I want to do that.
```
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = env("DATABASE_PROVIDER")
  url      = env("DATABASE_URL")
}
```
I'm using MySQL in development, and Postgres in production. Is there any way I can pass `mysql` or `postgresql` using the env variable?

**Comment by ryands17** [comment]
Hey @Shahzayb 👋 
Currently Prisma doesn't support passing the provider via an environment variable, so could you create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for the same.

Also I would like to understand your use case of using MySQL in development and Postgres in production.

**Comment by hi019** [comment, marked as answer]
This feature is currently implemented in the development branch. See #1487

**Reply by janpio** [reply]
The actually implemented solution is from https://github.com/prisma/prisma/issues/1487#issuecomment-635999202 and has already been shipped.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
